### PR TITLE
Updates to language tags and example requests

### PR DESCRIPTION
**Pull request by jakeost-msft**

### CODE REVIEW

**Review by angelgolfer-ms — COMMENTED** [review]
Hi @jakeost-msft , was there a specific reason you decided to use lower case for the region subtag?
The [overview](https://docs.microsoft.com/en-us/graph/api/resources/search-api-answers-overview) topic refers to RFC 4646. I suppose initially you're following the recommendations of [Section 2.1](https://docs.microsoft.com/en-us/graph/api/resources/search-api-answers-overview), which states even though these tag and subtag values are to be treated as case insensitive, the recommendation is consistent case formatting to help users in general. For that purpose and this recommendation - "all non-initial two-letter subtags are uppercase" - your initial treatment is appropriate.
Can you please clarify before I close on this PR?

**Comment by jakeost-msft** [comment]
> Hi @jakeost-msft , was there a specific reason you decided to use lower case for the region subtag? The [overview](https://docs.microsoft.com/en-us/graph/api/resources/search-api-answers-overview) topic refers to RFC 4646. I suppose initially you're following the recommendations of [Section 2.1](https://docs.microsoft.com/en-us/graph/api/resources/search-api-answers-overview), which states even though these tag and subtag values are to be treated as case insensitive, the recommendation is to apply consistent case formatting (consistent with RFC 4646 recommendations) to help users in general. For that purpose and the RFC recommendation - "all non-initial two-letter subtags are uppercase" - your initial treatment is appropriate. Can you please clarify before I close on this PR?

@angelgolfer-ms 
Yes, the reason for the change is because we are actually dependent on this formatting: [BingSupportedMarkets](https://office.visualstudio.com/OC/_search?action=contents&text=BingSupportedMarkets&type=code&lp=code-Project&filters=ProjectFilters%7BOC%7DRepositoryFilters%7BM365AdminServices%7D&pageSize=25&result=DefaultCollection/OC/M365AdminServices/GBmain//src/M365AdminServices/SearchSettingsService/Validation/ValidationConstants.cs)

I'm not sure if that is public facing or if we should change the wording regarding RFC 4646. I would appreciate any insight you can provide. Thanks!

File: api-reference/beta/resources/search-bookmark.md [hunk 1]
```diff
@@ -38,7 +38,7 @@ Inherits from [searchAnswer](../resources/search-searchAnswer.md).
 |categories|String collection|Categories commonly used to describe this bookmark. For example, IT and HR.|
 |availabilityStartDateTime|DateTimeOffset|Timestamp of when the bookmark will start to appear as a search result. Set as `null` for always available.|
 |availabilityEndDateTime|DateTimeOffset|Timestamp of when the bookmark will stop to appear as a search result. Set as `null` for always available.|
-|languageTags|String collection|A list of language names that are geographically specific and that this bookmark can be viewed in. Each language tag value follows the pattern {language}-{REGION}. As an example, `en-US` is English as used in the United States. See [supported language tags](search-api-answers-overview.md#supported-language-tags) for the list of possible values.|
+|languageTags|String collection|A list of language names that are geographically specific and that this bookmark can be viewed in. Each language tag value follows the pattern {language}-{REGION}. As an example, `en-us` is English as used in the United States. See [supported language tags](search-api-answers-overview.md#supported-language-tags) for the list of possible values.|
```
Comment (angelgolfer-ms) [inline api-reference/beta/resources/search-bookmark.md:41]: Line 41:
If you must require lower case for a region value, then please change {language}-{REGION} to {language}-{region}.

File: api-reference/beta/resources/search-qna.md [hunk 2]
```diff
@@ -37,7 +37,7 @@ Inherits from [searchAnswer](../resources/search-searchAnswer.md).
...
 |availabilityStartDateTime|DateTimeOffset|Timestamp of when the qna will start to appear as a search result. Set as `null` for always available.|
 |availabilityEndDateTime|DateTimeOffset|Timestamp of when the qna will stop to appear as a search result. Set as `null` for always available.|
-|languageTags|String collection|A list of language names that are geographically specific and that this QnA can be viewed in. Each language tag value follows the pattern {language}-{REGION}. As an example, `en-US` is English as used in the United States. See [supported language tags](search-api-answers-overview.md#supported-language-tags) for the list of possible values. |
+|languageTags|String collection|A list of language names that are geographically specific and that this QnA can be viewed in. Each language tag value follows the pattern {language}-{REGION}. As an example, `en-us` is English as used in the United States. See [supported language tags](search-api-answers-overview.md#supported-language-tags) for the list of possible values. |
```
Comment (angelgolfer-ms) [inline api-reference/beta/resources/search-qna.md:40]: Line 40:
If you must require lower case for a region value, then please change {language}-{REGION} to {language}-{region}.

File: api-reference/beta/resources/search-api-answers-overview.md [hunk 3]
```diff
@@ -25,46 +25,46 @@ The use cases for the APIs in this section involve managing administrative answe
 | Create, update, or delete a search answer        | [acronym](search-acronym.md), [bookmark](search-bookmark.md), [qna](search-qna.md) |
 
 ## Supported language tags
-A language tag of a [bookmark](search-bookmark.md) or [qna](search-qna.md) represents a geographically specific language in which that search answer (**bookmark** or **qna**) can be viewed. A language tag follows the pattern {language}-{REGION}. For more information on the pattern, see [RFC 4646](https://datatracker.ietf.org/doc/html/rfc4646).
+A language tag of a [bookmark](search-bookmark.md) or [qna](search-qna.md) represents a geographically specific language in which that search answer (**bookmark** or **qna**) can be viewed. A language tag follows the pattern {language}-{region}. For more information on the pattern, see [RFC 4646](https://datatracker.ietf.org/doc/html/rfc4646).
```
Comment (angelgolfer-ms) [inline api-reference/beta/resources/search-api-answers-overview.md:29]: Please see my message on Teams.

For line 28:
If you must require lower case for a region value, then please change the last sentence in line 28 to:
"For information on the supported values for {language} and {region}, see [RFC 4646](https://datatracker.ietf.org/doc/html/rfc4646)."

Then add the following in a separate paragraph:
"Note You must specify values for {language} and {region} in lower case."

**Review by angelgolfer-ms — CHANGES_REQUESTED** [review]
For interoperability based on RFC 4646, the best approach is to remove the requirement that the region subtag must be in lower case. After all, RFC 4646 states tags and subtags are case-insensitive.
If there are reasons you can't remove that requirement, then please see the comments I just added for additional changes in the docs.

**Review by angelgolfer-ms — APPROVED** [review]
Hi @jakeost-msft , I made an extra change to format the note in the API overview topic for effectiveness. I think this PR is good to be published. Please confirm so and set the label to ready to merge.

**Pull request merged**